freecad/trails/project/support/utils.py

```diff
@@ -280,3 +280,16 @@ def scrub_stationing(station):
         return -1
 
     return float(scrub)
+
+def rendering_fix(base):
```

Comment:
Can you explain how this works?

Reply:
When you created a new object at origin and moved it to directly to its original position that cause shifting between object.(like my points and surfaces we talked about before). So we need to create all object at the same coordinate near to working area(like local coordinate system). To use fonction you need to send base position of your object and it return a vector. You can subtract that value from your points coordinates and create your object near to 0,0,0 than move object placement to returned vector.

Reply:
I made a change to the comment.  I'll accept the function as it is, but I'm going to recommend you change the name to something more meaningful, like "compute_alternate_origin".  "rendering_fix" doesn't describe it very well.